scripts/devel/build_and_install.py

```diff
@@ -21,7 +21,7 @@
 from ros_buildfarm.argument import add_argument_build_tool
 from ros_buildfarm.argument import add_argument_build_tool_args
 from ros_buildfarm.common import Scope
-from ros_buildfarm.workspace import call_build_tool
+from ros_buildfarm.workspace import call_build_tool, call_abi_checker
```

Comment:
Nitpick: one import per line - in alphabetical order.

Reply:
cb68012